### DISCUSSION TITLE
Getting "Failed to resolve component.." when using Quasar without cli or umd

### DISCUSSION THREAD

**Discussion by yenniSuresh** [quasarframework/quasar#17505] [General - Components / Directives / etc] [answered]
Hi guys,

I am trying to upgrade from v1 to v2 (v2.16.9).

Not sure why but all the .vue files where i have quasar components are throwing error in browser 

`[Vue warn]: Failed to resolve component: q-btn
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.`

Seems all the components are not automatically registered globally anymore.

My application run on meteor framework and it has its own build process so i cannot use any cli or webpack or vite here. 

In my application am doing 

`import { Quasar } from 'quasar'`

`vueAppInstance.use(Quasar)`

Am i missing anything or its a bug?

**Comment by rstoenescu** [comment, marked as answer]
You need to explicitly import the components/directives/plugins/etc that you are using. Either globally (`app.use(Quasar, { ... }`) or in every vue file.
